mantle/cmd/kola/testiso.go

```diff
@@ -104,6 +106,31 @@ RequiredBy=multi-user.target
 RequiredBy=coreos-installer.target
 `, liveOKSignal)
 
+// TODO: replace unit with environment variables when available
+var preshimUnit = `[Unit]
+Before=coreos-installer.service
+[Service]
+Type=oneshot
+ExecStart=/bin/sh -c '/usr/bin/cp /usr/libexec/coreos-installer-service /usr/local/bin/'
+ExecStart=/bin/sh -c "/usr/bin/sed -i -r  's,coreos-installer (.*),coreos-installer \1 --offline,g' /usr/local/bin/coreos-installer-service"
+ExecStart=/bin/sh -c '/usr/bin/mount --bind /usr/local/bin/coreos-installer-service /usr/libexec/coreos-installer-service'
```

Comment:
Hmm, interesting approach. But... is that the right thing to do? We should be testing that the shipped code uses the offline mode by default, not just when coaxed into it. :) 

How about this: after the install is complete, we mount the rootfs partition, and sanity-check that the OSTree commit and build ID from `$rootmnt/.coreos-aleph-version.json` matches what we expect? That should cover what prompted this as well as any other issues of that nature.

Something like:

```
ExecStart=/usr/bin/mount /dev/disk/by-label/root /mnt
ExecStart=/usr/bin/jq -e '.["build"] == "<build>"' /mnt/.coreos-aleph-version.json
ExecStart=/usr/bin/jq -e '.["ostree-commit"] == "<commit>"' /mnt/.coreos-aleph-version.json
```

?